app/internal/email.py

```diff
@@ -177,3 +182,27 @@ def verify_email_pattern(email: str) -> bool:
         return True
     except EmailError:
         return False
+
+
+async def send_reset_password_mail(
+        user: ForgotPassword,
+        background_tasks: BackgroundTasks) -> bool:
+    """
+    This function sends a reset password email to user.
+    :param user: ForgotPassword schema.
+        Contains user's email address, jwt verifying token.
+    :param background_tasks: (BackgroundTasks): Function from fastapi that lets
+            you apply tasks in the background.
+    returns True
+    """
+    template = templates.get_template("reset_password_mail.html")
+    html = template.render(
+         recipient=user.username,
+         link=f"{CALENDAR_RESET_PASSWORD_PAGE}?token={user.token}",
```

Comment:
Maybe url_for?

Reply:
Is that even possible? I'm not sending a request here, its a link that is attached to an a-tag at the sent email template